pubpub/platform
# Add Number, NumericArray, StringArray schemas

## Issue(s) Resolved
Resolves #590 

## High-level Explanation of PR


This PR simply adds three new schemas: `Number`, `NumericArray`, and `StringArray`.

The new `Number` schema can be used in forms. It re-uses the existing `<TextElement />` input with `type="number"`. `NumericArray` and `StringArray` currently do not have any mapped inputs.

## Test Plan

- Add a new field to a community with the new `Number` schema
- Add two new fields: one using the `NumericArray` schema and one for `StringArray`
- Add these fields to a new form. Note that you can select an number input component for the field with the `Number` schema, but no component for the array components.
- Save the form and open the form for an existing pub. Change the value of the field with the backing `Number` schema. Confirm that the form persists the numeric value properly.

## Screenshots (if applicable)

## Notes
